circle.yml

```diff
@@ -121,6 +121,7 @@ defaults:
     run:
       name: "Test shared Hera"
       command: |
+        ASAN_OPTIONS=detect_leaks=1
```

Comment:
Do it only for WAVM otherwise it will stop on the first error in Binaryen or WABT.

Reply:
Aha, trying now.

Reply:
Also build WAVM with RelWithDebInfo: https://github.com/ewasm/hera/pull/420/files#diff-2a2be4bac1d9816c9bd26716619f9ed8R39